chia/types/eligible_coin_spends.py

```diff
@@ -299,16 +300,21 @@ async def process_fast_forward_spends(
                     )
                     # Nothing more to do for this spend, moving on
                     new_coin_spends.append(spend_data.coin_spend)
+                    new_bundle_coin_spends[coin_id] = BundleCoinSpend(
+                        coin_spend=spend_data.coin_spend,
+                        eligible_for_dedup=spend_data.eligible_for_dedup,
+                        eligible_for_fast_forward=spend_data.eligible_for_fast_forward,
+                        additions=spend_data.additions,
+                        cost=spend_data.cost,
+                    )
```

Comment:
```suggestion
                    new_bundle_coin_spends[coin_id] = copy.copy(spend_data)
```

If so, you could do it here too